src/CMakeLists.txt

```diff
@@ -214,7 +217,7 @@ vala_add(ayatana-indicator-sound-service
   accounts-service-sound-settings.vala
 )
 
-if(${HAVE_UT_ACCTSERVICE_PRIVACY_SETTINGS})
+if(${HAS_LOMIRI_ACCTSERVICE_PRIVACY_SETTINGS})
     vala_add(ayatana-indicator-sound-service
           accounts-service-system-sound-settings.vala
```

Comment:
This looks wrong. if PRIVACY_SETTINGS is defined, we vala_add ``accounts-service-system-sound-settings.vala``. And if SOUND_SETTINGS is defined, we vala_add ``accounts-service-privacy-settings.vala``???

I will merge this PR and file a follow up. Obviously this was broken all the time.